README.md

```diff
@@ -113,6 +113,7 @@ While we don't have our internal Style Guide, let's use the same as Stencil core
 Please add here, when you solve something you got stuck at.
 
 - In HTML, attributes need to use dash-case. In JSX you use camelCase.
+- When you add new components, it's best to restart build watcher because it usually breaks here.
```

Comment:
do we know why?

Reply:
I tried to use some time to figure it out but couldn't in that time.